src/d3a/d3a_core/simulation.py

```diff
@@ -388,7 +388,8 @@ def _execute_simulation(self, slot_resume, tick_resume, console=None):
         self.deactivate_areas(self.area)
         self.simulation_config.external_redis_communicator.\
             publish_aggregator_commands_responses_events()
-
+        if self.simulation_config.external_connection_enabled:
```

Comment:
Non-market areas should not publish these events AFAIS.
I propose to move the `publish_event_finish_myco` and `publish_event_tick_myco` and `publish_event_market_cycle_myco ` into the area and check whether the strategy is None. 

Also as a side note, in the future we might have to think about only sending these events for areas that have an active myco client connected to them. But I guess for these initial cases, we can leave it as is.

Reply:
I'm publishing ticks / markets from the simulation not from areas
Also I contacted Colin and he mentioned that we will only allow 1 myco client to get open offers/bids and post recommendations (and he did not deny that they may leave it open for clients to compete) so he also insisted that I should not put constraints now because they need more time to agree on the process of registration, this is why I;'m only validating if the client is loggedin

Reply:
You mixed something up here with my comment on d3a-web.

Here I am just saying that it does not make sense to send the e.g. `publish_event_market_cycle_myco` for a load. Only for a market it makes sense. And This is why i am advocating to move this into the area where you have access to the check `if area.strategy` (if it is None, then it is a market area).

Reply:
@hannesdiedrich you're missing the fact the the Myco client is not aware of areas, it is for the whole simulation
pay attention to the difference between it and the api-client , here the client subscribes to the simulation channel whereas in the api-client we used to instantiate objects for each area where it subscribes to this specific area events

Reply:
🤦  I am sorry. You are right of course. If this is settled, then some of my other comments dont make sense as well. Let me re-review. Thanks for bering with me!